CONTRIBUTING.md

```diff
@@ -84,84 +91,129 @@ refer to the [style guide](contributing-guides/style-guide.md).
+Of course, this shouldn't sacrifice content clarity, such as when documenting tools where this terminology has specific technical meanings and its usage is central to explaining the involved concepts.
 
 ## Submitting a pull request
 
-The easiest way to submit a change is to just edit the page directly on the GitHub interface.
+### Testing pages locally
+
+Once you have written a `tldr` page, you can test its syntax locally using [`tldr-lint`](https://github.com/tldr-pages/tldr-lint).
+
+The latest version of [NodeJS](https://nodejs.org) is required to install `tldr-lint` with the following command:
+
+```sh
+npm install --global tldr-lint
+```
+
+Once its installed, you can test your page by running the following command:
+
+```sh
+tldr-lint <path/to/page.md>
+```
+
+Now, you are ready to submit a pull request!
+
+> [!TIP]
+> Additionally, inside the `tldr` directory you can install the dependencies using `npm install` command and now when you commit your changes, the tests will run automatically via the pre-commit hook.
+
+### Submitting changes
+
+The easiest way to submit a change is to edit the page directly on the GitHub interface.
+
 Check out the step-by-step instructions (with screenshots) on
 [GitHub Help](https://help.github.com/articles/editing-files-in-another-user-s-repository/).
 
 Alternatively, you can do most of the process
 [using Git on the command-line](contributing-guides/git-terminal.md).
 
+> [!TIP]
+> After creating a pull request, it is suggested to enable the "Allow edits by maintainers" option (This only needs to be done once the first time you create a PR). It allows maintainers to make changes to your pull request and assist you in getting it merged.
+
 ### Accepting suggestions within a pull request
 
-The easiest way to apply suggested changes is to accept the suggestion made on your pull request. Refer to the [GitHub docs](https://docs.github.com/en/enterprise-server@3.2/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) for more details.
+The easiest way to apply suggested changes is to accept the suggestion made on your pull request. Refer to the [GitHub docs](https://docs.github.com/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) for more details.
 
-To commit a suggestion to your pull request, simply click on `Commit suggestion`:
+To commit a suggestion to your pull request, click on `Commit suggestion`:
 
 ![Commit suggestion button in Github](./images/commit-suggestion-button.png)
 
+When you want to commit multiple suggestions, go to the "Files changed" tab and batch all suggestions. Now, click `Commit suggestions` button and enter a commit message to create a single commit.
```

Comment:
Perhaps this will read better?
```suggestion
If you want to commit multiple suggestions, go to the "Files changed" tab and batch all suggestions. Now, click `Commit suggestions` button and enter a commit message to create a single commit.
```